### DISCUSSION TITLE
CSS issue with group button / autocomplete button, etc.. with "left-radius"

### DISCUSSION THREAD

**Discussion by Cyril30** [primefaces/community#600] [PrimeVue] [answered]
Hello, 

I have an issue with the button attached to field or other buttons.. 
![image](https://github.com/primefaces/community/assets/35692543/53c5dfc6-ed85-4ed8-8c41-8713e10505ce)

As an example with autocomplete, the following code is crossed / not used when I inspect the element while debugging.
`.p-autocomplete-dd .p-autocomplete-dropdown {
    border-top-left-radius: 0;
    border-bottom-left-radius: 0px;
}`

I tried to overload the CSS using passtrought option. Here is an example : 
` <AutoComplete id="client" :dropdown="true" :multiple="true" `
                   `    :pt="{`
                     `    loadingIcon: 'backred-0',`
                     `    dropdownButton: 'backred-0'`
               ` }" />`

my backred class add the background color red, as you can see in the image for the loading image : 
![image](https://github.com/primefaces/community/assets/35692543/a2917cb6-f874-49d8-8cbb-0c85c5a887ff)

**It seems that the p-button component always have the priority over** the p-autocomplete-dd  or p-autocomplete-dropdown, or group and all of these.

Do you have any idea about why I have this issue ? 

Thank you very much, have a nice day!

Cyril

**Comment by tugcekucukoglu** [comment, marked as answer]
Hello,

This part is still open to development. Currently, the team is working on a new preset. We'll try to fix this kind of defect.

Thanks for the feedback.
